utils/hare-bootstrap

```diff
@@ -288,39 +293,41 @@ jq '[.[] | {key, value: (.value | @base64)}]' < $conf_dir/consul-kv.json |
     consul kv import - > /dev/null
 echo ' OK'
 
-say 'Starting Consul on other nodes...'
-pids=()
-while read node bind_ip; do
-    ssh $node "PATH=$PATH $(which mk-consul-env) \
-                          --mode server --bind $bind_ip --join $join_ip &&
-               sudo systemctl start hare-consul-agent" &
+if [ "$skip_consul_start" = false ] ; then
+    say 'Starting Consul on other nodes...'
+    pids=()
+    while read node bind_ip; do
+        ssh $node "PATH=$PATH $(which mk-consul-env) \
+                              --mode server --bind $bind_ip --join $join_ip &&
+                   sudo systemctl start hare-consul-agent" &
     pids+=($!)
-done < <(get_server_nodes | grep -vw $(node-name) || true)
+    done < <(get_server_nodes | grep -vw $(node-name) || true)
 
-while read node bind_ip; do
-    ssh $node "PATH=$PATH $(which mk-consul-env) \
-                          --mode client --bind $bind_ip --join $join_ip &&
-               sudo systemctl start hare-consul-agent" &
-    pids+=($!)
-done < <(get_client_nodes)
-wait4 ${pids[@]-}
-agents_nr=$(( ${#pids[@]} + 1 ))
+   while read node bind_ip; do
+        ssh $node "PATH=$PATH $(which mk-consul-env) \
+                              --mode client --bind $bind_ip --join $join_ip &&
+                   sudo systemctl start hare-consul-agent" &
+        pids+=($!)
+    done < <(get_client_nodes)
+    wait4 ${pids[@]-}
+    agents_nr=$(( ${#pids[@]} + 1 ))
 
-# Waiting for the Consul agents to get ready...
-count=1
-while (( $(get_ready_agents_nr) != $agents_nr )); do
-    if (( $count > 5 )); then
-        echo 'Some agent(s) failed to start in due time:' >&2
-        diff <(get_ready_agents | sort) \
-             <(get_all_nodes | awk '{print $1}' | sort) | sed 1d >&2
-        echo 'Check connectivity and firewall (Consul ports must be opened)' >&2
-        exit 1
-    fi
-    echo -n '.'
-    sleep 1
-    (( count++ ))
-done
-echo ' OK'
+    # Waiting for the Consul agents to get ready...
+    count=1
+    while (( $(get_ready_agents_nr) != $agents_nr )); do
```

Comment:
Let's have this check unconditional to `skip_consul_start` value. Because even though Consul processes are running we need to make sure that all are running if not the we must error out.